src/pages/EventsPage/EventsPage.jsx

```diff
@@ -1,27 +1,74 @@
 import React, { useEffect, useState } from "react";
 import "./EventsPage.css"
 import { useFirebase } from "../../context/Firebase";
+import Events from "../../components/Events/Events";
+import {motion} from "framer-motion";
+import Button from "../../components/UI/button/Button";
+
 
 const EventsPage = () => {
   const firebase = useFirebase();
-  const [eventData, seteventData] = useState([]);
+  // const [eventData, seteventData] = useState([]);
+  const [eventType,setEventType] = useState("events")
+  const buttonStyle = {
+    height:"43px",
```

Comment:
this buttonStyle is not required pls remove it

Reply:
ok